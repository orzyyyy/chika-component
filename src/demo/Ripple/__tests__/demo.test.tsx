import React from 'react';
import { mount } from 'enzyme';
import 'nino-cli/scripts/setup';
import RippleDemo from '..';

describe('RippleDemo', () => {
  it('render correctly', () => {
    const wrapper = mount(<RippleDemo />);
    expect(wrapper).toMatchSnapshot();
  });
});
