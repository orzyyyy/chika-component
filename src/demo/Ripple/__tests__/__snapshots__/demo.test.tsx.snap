// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RippleDemo render correctly 1`] = `
<RippleDemo>
  <div
    className="Ripple_demo"
  >
    <div
      className="item"
      key="item0"
    >
      <div
        className="item-label"
      >
        test1
      </div>
      <Ripple
        wrapWidth={0}
      >
        <div>
          <div
            className="Ripple"
            style={Object {}}
          />
        </div>
      </Ripple>
    </div>
    <div
      className="item"
      key="item1"
    >
      <div
        className="item-label"
      >
        test2
      </div>
      <Ripple
        wrapWidth={0}
      >
        <div>
          <div
            className="Ripple"
            style={Object {}}
          />
        </div>
      </Ripple>
    </div>
  </div>
</RippleDemo>
`;
